index.html

```diff
@@ -381,6 +391,71 @@ <h2 class="contact-heading">Contact me</h2>
         </div>
       </section>
     </div>
-    <script defer src="./index.js" type="module"></script>
+    <script defer src="./script/index.js" type="module"></script>
+
+    <!-- General Card for Each popup Window  -->
+    <div class="wrapper">
+      <div id="card1" class="card-works popup-card">

```

Comment:
- [ ] please try to make the pop-up responsive in the mobile version and in the desktop version and try to remove the scroll bar and all the content of popUp should be displayed 
![image](https://user-images.githubusercontent.com/61790587/173922119-587d1bf0-3be6-4ecd-9c23-a49d20c30feb.png)
![image](https://user-images.githubusercontent.com/61790587/173922132-f1b7070c-a058-4bdf-985a-d005988753c8.png)